src/util-file.c

```diff
@@ -149,6 +155,25 @@ void FileForceTrackingEnable(void)
     g_file_force_tracking = 1;
 }
 
+void FileWriteMetaDisable(void)
```

Comment:
probably should not be a global at all to make it easier to support multiple instances of file-store (if that doesn't already work)